src/hlstrafe.cpp

```diff
@@ -498,6 +498,30 @@ namespace HLStrafe
 		avec[1] = wishvel[1];
 	}
 
+	void UpdateLookAtViewangle(const PlayerData& player, CurrentState& curState)
+	{
+		float target[3];
+		target[0] = curState.TargetYawLookAtOrigin[0] + curState.Parameters.Parameters.LookAt.X;
+		target[1] = curState.TargetYawLookAtOrigin[1] + curState.Parameters.Parameters.LookAt.Y;
+		target[2] = curState.TargetYawLookAtOrigin[2] + curState.Parameters.Parameters.LookAt.Z;
+
+		float difference[3];
+		VecSubtract<float, float, 3>(target, player.Origin, difference);
+
+		double yaw = 0;
+		double pitch = 0;
+		if (difference[0])
```

Comment:
```suggestion
		if (difference[0] || difference[1])
```
I believe this is the right form?